### ISSUE TITLE
Delete Uploads Error: Call to a member function getParentFolder() on array

### ISSUE THREAD

**Issue by icehawk-de** [svewap/form_extended#20]
I use the modified v12 version of multi_upload -> https://github.com/WapplerSystems/form_extended/pull/19

It works as intended, but if you use the finisher "Delete Uploads" you get an FE error:
Call to a member function getParentFolder() on array

Everything else works: Emails are generated with attachments. No images are saved on Server.

**Comment by darthnorman** [comment]
Yeah, the finisher doesn't work when you are using form_extended. DeleteUploads needs to be overwritten as well.
